spec/models/classroom_spec.rb

```diff
@@ -49,5 +56,39 @@
       subject.user = create(:student)
       expect(subject).not_to be_valid
     end
+
+    it 'has end time earlier than start time' do
+        subject.start_time = Time.now + 1.hour
+        subject.end_time = Time.now 
+        expect(subject).not_to be_valid
+    end
+
+    it 'without a start time' do
+      subject.start_time = nil
+      expect(subject).not_to be_valid
+    end
+
+    it 'without an end time' do
+      subject.end_time = nil
+      expect(subject).not_to be_valid
+    end
+
+    it 'without start and end time' do
```

Comment:
I'm not quite sure if this was necessary but I figured why not add it, so I did ^^;